repos/system_upgrade/common/actors/satellite_upgrade_services/actor.py

```diff
@@ -0,0 +1,35 @@
+import glob
+import os
+
+from leapp.actors import Actor
+from leapp.models import SatelliteFacts
+from leapp.tags import ApplicationsPhaseTag, IPUWorkflowTag
+
+SYSTEMD_WANTS_BASE = '/etc/systemd/system/multi-user.target.wants/'
+SERVICES_TO_DISABLE = ['dynflow-sidekiq@*', 'foreman', 'foreman-proxy',
+                       'httpd', 'postgresql', 'pulpcore-api', 'pulpcore-content',
+                       'pulpcore-worker@*', 'tomcat']
+
+
+class SatelliteUpgradeServices(Actor):
```

Comment:
this is just moved code from `satellite_upgrade_data_migration/actor.py` as those steps are el-version-agnostic, while the code remaining in `satellite_upgrade_data_migration/actor.py` is only needed for 7to8 due to the SCL usage